### DISCUSSION TITLE
deploy on Fly.io

### DISCUSSION THREAD

**Discussion by brunowilliang** [appwrite/appwrite#4833] [General]
Hi guys, can anyone guide me step by step to deploy on fly.io?

**Comment by brunowilliang** [comment]
Hi guys, can anyone help me create a dockerfile to deploy on fly.io?

**Comment by neiljaywarner** [comment]
also interested, especially if no free heroku tier or gcp tier that actually works.

**Comment by been-there-done-that** [comment]
+1
